codeforamerica/vita-min
# FYST-441 Create DF Instakes for NC

## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-441
## Is PM acceptance required? 
- Yes - don't merge until JIRA issue is accepted!

## What was done?
Created 4 new df xmls for NC for single, hoh, qss and mfs

## How to test?
Use heroku link to check the DF intakes for each new persona and check for filing statuses and success for submissions.
